src/main/app/case/case.ts

```diff
@@ -71,6 +72,13 @@ export interface Case {
   livingInEnglandWalesSixMonths?: YesOrNo;
```

Comment:
I know this isn't your code but you can't have a type here. This index is a mapping of one to one fields between the API and the frontend